gflownet.py

```diff
@@ -782,6 +805,7 @@ def flowmatch_loss(self, it, batch):
         else:
             # TODO: potentially mask invalid actions next_q
             next_q = self.model(sp)
+        next_q = torch.where(masks == 0, next_q, -loginf)
```

Comment:
Would it not be `masks == 1`? We want to make `-loginf` the invalid (mask == True) actions.

Reply:
Sorry, got `torch.where` wrong. All good!